README.md

```diff
@@ -70,6 +70,11 @@ Or by setting environment variable `BENTOML_DO_NOT_TRACK=True`:
 export BENTOML_DO_NOT_TRACK=True
 ```
 
+### Contributers! ###
```

Comment:
```suggestion
### Contributors! ###
```

Reply:
Also I thinkk this section should be under LICENSE

Reply:
I only looked at one example here: https://github.com/nushell/nushell

and they had it above license

Reply:
cool, I don't have too big of an opinion on this. But theres a typo :)

Reply:
nice! thanks